SmartHomeManager/SmartHomeManager.View/src/components/Analytics/UsageBar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Chart from "chart.js/auto";
+import { Bar } from "react-chartjs-2";
+import {
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+  Box,
+  Center,
+  Container,
+  Text,
+  Heading
+} from '@chakra-ui/react'; 
+
+const labels = ["January", "February", "March", "April", "May", "June"];
+    const data = {
+      labels: labels,
+      datasets: [
+        {
+          label: "My First dataset",
+          backgroundColor: "rgb(255, 99, 132)",
+          borderColor: "rgb(255, 99, 132)",
+          data: [27, 10, 5, 2, 20, 30, 45],
+        },
+      ],
+    };
+function UsageBar() {
+    
+
+    return (
+      <Container maxWidth={1600} mt={2} centerContent>
```

Comment:
@rawsashimi1604 remove maxwidth and centercontent

Reply:
@junweilam @jianweiiii please fix under branch `fix/analysis-front-end`, submit PR when ready

Reply:
@rawsashimi1604 fixed